app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,41 @@
+class RestaurantsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :set_restaurant, only: [:show, :edit, :update, :destroy]
+
+  def index
+    render json: {restaurants: ActiveModel::SerializableResource.new(Restaurant.sorted_by_name, each_serializer: RestaurantSerializer) }
```

Comment:
GJ with the serializers.
Minor: in this case of an extra long line its always good to extract to a variable.